src/kudu/consensus/proxy_policy.h

```diff
@@ -0,0 +1,44 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#pragma once
+
+namespace kudu {
+namespace consensus {
+
+enum class ProxyPolicy {
+  // Proxy is disabled and the leader ships ops to all peers directly
+  DISABLE_PROXY = 1,
+  // Proxy routing is built implicily using current active raft configs.
+  // One peer is choosen to act as a 'proxy peer' in a given region.
+  // The rules used to build proxy topology in this policy is:
+  // 1. A given region has at-most one valid proxy peer.
```

Comment:
This is fine, but we will need to extend this to handle multiple potential proxy peers in the future.

Reply:
Yes, have some ideas - but not needed immediately. For example,  we can have a list of 'valid proxy peers' and simply round-robin between them.